### DISCUSSION TITLE
hosting orchard behind nginx as reverse proxy and http://domain.com/backend  lead to 404

### DISCUSSION THREAD

**Discussion by ricocsharp** [OrchardCMS/OrchardCore#12799] [General]
I'm hosting orchard behind nginx as reverse proxy,  my url is http://domain.com/backend .  I'm redirected to http://domain.com/Login.aspx which leads to 404 error is there any way to make this site work behind this url: http://domain.com/backend so the login should be http://domain.com/backend/Login

**Comment by jtkech** [comment]
You can do it yourself but we have a `ReverseProxy` module to enlist the needed middleware and to define how to apply what is forwarded (what is forwarded may depend on your nginx config).

**Reply by ricocsharp** [reply]
Thanks @jtkech I already saw the ReverseProxy  module and even with it I had the issue, I finally solved it using this piece of code in startup:
if (env.IsProduction())
            {
                app.Use((context, next) =>
                {
                    context.Request.PathBase = "/backend";
                    return next();
                 });
            }
then I removed extra conf from nginx and now everething is ok 

Thanks a lot
bests

**Reply by hishamco** [reply]
How the above code fixes the issue?!!

> then I removed extra conf from nginx and now everething is ok

What are configuration that been removed, may be it's useful to have a docs about using NGinx with OC
